googler

```diff
@@ -2253,16 +2249,35 @@ def self_upgrade(include_git=False):
 # Miscellaneous functions
 
 def https_proxy_from_environment():
-    if 'https_proxy' not in os.environ:
-        return None
-    proxy = urllib.parse.urlparse(os.environ['https_proxy']).netloc
-    # urlparse recognizes a netloc only if it is introduced by '//'
-    # (https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse).
-    # This means a bare host:port like 'localhost:8118' won't be
-    # recognized. In that case, try again with the '//' prefix.
-    if not proxy:
-        proxy = urllib.parse.urlparse('//' + os.environ['https_proxy']).netloc
-    return proxy if proxy else None
+    return os.getenv('https_proxy')
```

Comment:
A nice side effect is that the `https_proxy` env var can be used directly without separate parsing.